printer_mods/OboMaker3D/C17-EXT-CARTRIDGE/Readme.md

```diff
@@ -0,0 +1,54 @@
+# C17 -EXT-CARTRIDGE, HEPA Carbon filter
+
+![C17 -EXT-CARTRIDGE, HEPA Carbon filter](JPG/IMG_1.png "")
+
+![C17 -EXT-CARTRIDGE, HEPA Carbon filter](JPG/IMG_2.png "C17 -EXT, External HEPA Carbon filter")
+
+The C17-EXT-CARTRIDGE is designed to be used with the C17-EXT HEPA Carbon filter.
+
+This is a first publication, although all parts has been tested, there may be errors do not hesitate to contact me.
+
+## Assembly Manual
+
+[![Assembly Manual](JPG/IMG_3.png)](Doc/VORON_C17_EXT_CARTRIDGE_V1_0.pdf "Assembly Manual")
+
+# BOMs
+
+The list of parts is described in the manual.
+
+## HEPA 13 Filter
+
+Use HEPA 13 filters from 10mm to 18mm thick. The best is to find a HEPA 13 filter for vacuum cleaner and cut it with a cutter. The references change according to the country.
+
+I tested this one after cutting: 
+
+- The best but I do not find the reference in other countries: [Amazon.fr](https://www.amazon.fr/gp/product/B01MCTSXVR/ref=ppx_yo_dt_b_asin_title_o06_s00?ie=UTF8&psc=1)
```

Comment:
please shorten to `https://www.amazon.fr/gp/product/B01MCTSXVR` the rest is just tracking data.